engine/requirements.txt

```diff
@@ -10,7 +10,7 @@ redis==3.4.1
 humanize==0.5.1
 uwsgi==2.0.21
 django-cors-headers==3.7.0
-django-debug-toolbar==3.2.1
```

Comment:
bumped for this reason:
```bash
```bash
apps/twilioapp/tests/test_phone_calls.py: 66 warnings
  /usr/local/lib/python3.11/site-packages/debug_toolbar/utils.py:255: DeprecationWarning: currentThread() is deprecated, use current_thread() instead
    thread = threading.currentThread()
```
```